frida_tools/application.py

```diff
@@ -445,6 +451,47 @@ def work():
 
         return result[0]
 
+    def return_real_args(self, parser):
+        real_args = sys.argv[1:]
+        performPass = True
+        files_processed = set()
+
+        while (performPass):
+            offset = self.find_arg_file_offset(real_args, parser)
+            if (offset < 0):
+                performPass = False
+                continue
+
+            file_path = real_args[offset+1]
+            
+            if file_path in files_processed:
+                parser.error('File "' + file_path + '" given twice as -O argument')
+
+            if os.path.isfile(file_path):
+                with codecs.open(file_path, 'r', 'utf-8') as f:
+                  new_arg_text = f.read()
+            else:
+                parser.error('File "' + file_path + '" following -O option is not a valid file')
```

Comment:
Strings should be double-quoted. Better to use `.format()` here.

Reply:
Done